extensions/typescript-language-features/src/languageFeatures/diagnostics.ts

```diff
@@ -148,6 +151,76 @@ class DiagnosticSettings {
 	}
 }
 
+class DiagnosticsTelemetryManager extends Disposable {
```

Comment:
Not seeing where this class is currently used in the PR

Reply:
It's used on line 244 where provided you fall in the category of users who send diagnostics, an instance of DiagnosticsTelemetryManager is instantiated